randovania/gui/data_editor.py

```diff
@@ -240,14 +242,15 @@ def update_game(self, game: GameDescription):
             self.focus_on_connection(current_connection)
 
     @classmethod
-    def open_internal_data(cls, game: RandovaniaGame, edit_mode: bool) -> Self:
+    def open_internal_data(cls, game: RandovaniaGame, edit_mode: bool) -> DataEditorWindow:
```

Comment:
could change the return to use `cls` instead of explicitly using `DataEditorWindow`